src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -62,37 +63,46 @@
                     :rules="orgNameRules"
                   />
                 </div>
+
+                <!-- Roles -->
                 <label class="sub-header">Roles</label>
                 <v-row>
                   <v-col cols="4" v-if="isPerson">
-                    <div :class="{'highlightedRole': isRoleLocked(Roles.COMPLETING_PARTY)}">
+                    <div :class="{ 'highlightedRole': isRoleLocked(Roles.COMPLETING_PARTY) }">
                       <v-checkbox
-                        v-model="isCompletingParty"
-                        id='cp-checkbox'
-                        label="Completing Party"
+                        id="cp-checkbox"
+                        v-model="selectedRoles"
```

Comment:
All 3 checkboxes have the same v-model property (an array), and different "values".

Reply:
This allows the role rules to be simplified (and also, the rules didn't update properly with 3 different v-model properties).